packages/manager/src/features/CloudPulse/Alerts/CreateAlert/Criteria/DimensionFilterValue/DimensionFilterAutocomplete.tsx

```diff
@@ -64,28 +20,35 @@ export const DimensionFilterAutocomplete = (
     multiple,
     name,
     fieldOnChange,
-    values,
     disabled,
     fieldOnBlur,
-    isError,
-    isLoading,
     placeholderText,
     errorText,
     fieldValue,
+    serviceType,
+    dimensionLabel,
+    values,
   } = props;
 
+  const options = useMemo(
+    () =>
+      getStaticOptions(
+        serviceType ?? undefined,
```

Comment:
```suggestion
        serviceType,
```

lets convert the serviceType parameter to null instead of undefined here. Same undefined to null conversion is happening so its better keep it null instead of undefined [utils.ts](https://github.com/ACLPManager/manager/blob/44ac5ec199296e2fd1c4c9abb0a953f4a3de639e/packages/manager/src/features/CloudPulse/Alerts/CreateAlert/Criteria/DimensionFilterValue/utils.ts#L81)